boards/openmote-cc2538/include/periph_conf.h

```diff
@@ -82,6 +82,13 @@
 /** @} */
 
 /**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (8)
```

Comment:
Does this refers to the number of enabled ADC, or the maximum possible? as normally PA0/PA1 are used as UART0 perhaps this should be less.  Also you only seem to define three ADx_PINs.